coursemapper-kg/concept-map/src/pipelines/expand_material.py

```
@@ -23,14 +25,14 @@
 
         # Expand material graph
-        self.expand_material_graph(graph, push_log_message)
+        self.expand_material_graph(graph, push_log_message, graph_db)
 
         # Save subgraph
         if graph_db is not None:
             graph_db.save_graph(graph)
             self.gcn(material_id, graph_db)
 
-    def expand_material_graph(self, graph: Graph, push_log_message: Callable):
+    def expand_material_graph(self, graph: Graph, push_log_message: Callable, graph_db: GraphDB):
```

Ruff (0.8.2)

`expand_material_graph` is too complex (15 > 10)

(C901)